resources/jscomposition/cases/casesDetail/components/EllipsisMenu.vue

```diff
@@ -1,54 +1,49 @@
 <template>
-  <td class="tw-sticky tw-w-0 tw-right-0 tw-top-0">
-    <div class="tw-float-right tw-relative tw-right-4">
-      <button
-        v-for="item in menuOptions"
-        :key="item.menu"
-        type="button"
-        class="tw-rounded-lg tw-bg-white tw-px-2 tw-py-1 tw-text-xs tw-font-semibold tw-text-gray-600 tw-shadow-sm
+  <div class="tw-float-right tw-absolute tw-right-4 tw-top-0">
+    <button
+      v-for="item in menuOptions"
+      v-if="row.screenId"
```

Comment:
this must be in different tags
https://vuejs.org/style-guide/rules-essential.html#avoid-v-if-with-v-for